src/localization/locales/fi.json

```
@@ -114,6 +116,7 @@
     "logo": "Logo",
     "disableProjectTheme": "Poista projektiteema käytöstä",
     "dropzoneText": "Tiputa tai klikkaa lisätäksesi tiedoston",
+    "uploadWarningNoFile": "Virheellinen tiedosto: Ei tiedostoa ladattu.",
```

Is this a message that appears when there is no file to upload? Then it could be simple like this:
```suggestion
    "uploadWarningNoFile": "Virheellinen tiedosto: Ei tiedostoa.",
```